packages/react/src/utilities/hooks/useCheckboxGroup/useCheckboxGroup.ts

```diff
@@ -60,7 +60,8 @@ export type GetCheckboxProps = Omit<
 > & {
   /** Enables indeterminate handling for this `Checkbox` and `CheckboxGroup` */
   allowIndeterminate?: boolean;
-  ref?: React.Ref<HTMLInputElement | null>;
+  ref?: React.ForwardedRef<HTMLInputElement>; // Use this to match Ref from `Checkbox`, remove when `Checkbox` no longer uses `forwardRef`
```

Comment:
Just curious - did we test that this is also backwards compatible? 🤓

Reply:
Jepp, tested with `18.3.1`, which is our lowest supported/wanted version.